rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_wind_plugin.h

```diff
@@ -57,6 +58,9 @@ static constexpr double kDefaultWindSpeedVariance = 0.0;
 static const math::Vector3 kDefaultWindDirection = math::Vector3(1, 0, 0);
 static const math::Vector3 kDefaultWindGustDirection = math::Vector3(0, 1, 0);
 
+static constexpr bool kDefaultCustomStaticWindField = false;
+static const std::string kDefaultCustomWindFieldPath = "$(find rotors_gazebo)";
```

Comment:
Indeed, it does not. Since this default value was not being used meaningfully anyways, I simply deleted it.